cmd/system-probe/modules/network_tracer.go

```diff
@@ -193,14 +193,12 @@ func (nt *networkTracer) Register(httpMux *module.Router) error {
 			maps = strings.Split(listMaps, ",")
 		}
 
-		ebpfMaps, err := nt.tracer.DebugEBPFMaps(maps...)
+		err := nt.tracer.DebugEBPFMaps(w, maps...)
```

Comment:
who is responsible to write the output as json?

Reply:
In a previous discussion (https://github.com/DataDog/datadog-agent/pull/21598#discussion_r1430458762) Bryce said that it isn't needed to output as JSON, as in this case it's just serializing a string, the output will be the string but with quotes, that's why I removed the JSON call.